src/test/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/PopulateUpdateSubmissionTest.java

```diff
@@ -0,0 +1,210 @@
+  @Mock private TrustCorporateValidator trustCorporateValidator;
+
+  {
+    beneficialOwnerIndividualDtoList.add(
+        BeneficialOwnerAllFieldsMock.getBeneficialOwnerIndividualDto());
+  }
+
+  {
+    beneficialOwnerCorporateDtoList.add(
+        BeneficialOwnerAllFieldsMock.getBeneficialOwnerCorporateDto());
+  }
+
+  {
+    beneficialOwnerGovernmentOrPublicAuthorityDtoList.add(
+        BeneficialOwnerAllFieldsMock.getBeneficialOwnerGovernmentOrPublicAuthorityDto());
+  }
+
+  {
+    managingOfficerIndividualDtoList.add(ManagingOfficerMock.getManagingOfficerIndividualDto());
+  }
+
+  {
+    managingOfficerCorporateDtoList.add(ManagingOfficerMock.getManagingOfficerCorporateDto());
+  }
+
+  {
+    trustDataDtoList.add(TrustMock.getTrustDataDto());
+  }
+
+  @Test
+  void testPopulateUpdateSubmissionJSON() throws Exception {
+    output = getJsonOutput();
+    assertEquals("{", output.substring(0, 1));
+    assertEquals("}", output.substring(output.length() - 1));
+  }
+
+  @Test
+  void testPopulateUpdateSubmissionForPresenterJSON() throws Exception {
+    output = getJsonOutput();
+    assertTrue(output.contains(PRESENTER_JSON));
+  }
+
+  @Test
+  void testPopulateUpdateSubmissionForDueDiligenceJSON() throws Exception {
+    output = getJsonOutput();
+    assertTrue(output.contains(DUE_DILIGENCE_JSON));
+  }
+
+  @Test
+  void testPopulateUpdateSubmissionForUserSubmissionJSON() throws Exception {
+    output = getJsonOutput();
+    assertTrue(output.contains(USER_SUBMISSION_JSON));
+  }
+
+  @Test
+  void testPopulateUpdateSubmissionForBeneficialOwnerJSON() throws Exception {
+    output = getJsonOutput();
+    assertTrue(output.contains(BENEFICIAL_OWNER_JSON));
+  }
+
+  @Test
+  void testPopulateUpdateSubmissionForBosOrMosAddedOrCeasedJSON() throws Exception {
+    output = getJsonOutput();
+    assertTrue(output.contains(ANY_BOS_OR_MOS_ADDED_OR_CEASED_JSON));
+  }
+
+  private String getJsonOutput() throws Exception {
+
+    setIsTrustWebEnabledFeatureFlag(true);
+    buildOverseasEntitySubmissionDto();
+    overseasEntitySubmissionDto.setDueDiligence(dueDiligenceDto);
+    Errors errors =
+        overseasEntitySubmissionDtoValidator.validateFull(
+            overseasEntitySubmissionDto, new Errors(), LOGGING_CONTEXT);
+    verifyValidateFull();
+    verify(ownersAndOfficersDataBlockValidator, times(1))
+        .validateOwnersAndOfficersAgainstStatement(eq(overseasEntitySubmissionDto), any(), any());
+    PopulateUpdateSubmission populateUpdateSubmission =
+        new PopulateUpdateSubmission(overseasEntitySubmissionDto, new UpdateSubmission());
+    UpdateSubmission updateSubmission = populateUpdateSubmission.populate();
+    ObjectMapper objectMapper = new ObjectMapper();
+
+    objectMapper = JsonMapper.builder().findAndAddModules().build(); //
+    output = objectMapper.writeValueAsString(updateSubmission);
+    return output;
+  }
+
+  private void verifyValidateFull() {
```

Comment:
Not sure why we are checking these validators have been called. They are not part of this PR?